x-pack/package.json

```diff
@@ -262,13 +262,14 @@
     "mapbox-gl": "0.54.0",
     "mapbox-gl-draw-rectangle-mode": "^1.0.4",
     "markdown-it": "^8.4.1",
+    "markdown-it-link-attributes": "^2.1.0",
+    "memoize-one": "^5.0.0",
     "mime": "^2.2.2",
     "mkdirp": "0.5.1",
     "moment": "^2.20.1",
     "moment-duration-format": "^1.3.0",
     "moment-timezone": "^0.5.14",
-    "memoize-one": "^5.0.0",
-    "monaco-editor": "^0.17.0",
+    "monaco-editor": "^0.14.3",
```

Comment:
This looks to be downgrading `monaco-editor` from `^0.17.0` to `^0.14.3`, I don't think this is expected.